### ISSUE TITLE
After update to 4.1.0 Android build breaks: resource drawable/icon (aka org.nativescript.mydrawerng:drawable/icon) not found

### ISSUE THREAD

**Issue by ParVisual** [NativeScript/nativescript-cli#3673]
Nativescript 4.1.0
Component tns-core-modules 4.1.0
Component tns-android 4.1.2

Angular CLI: 6.0.1
Node: 8.11.1
OS: win32 x64
Angular: 5.2.11

Android SDK:  Fully updated.

Error: 

> D:\Proyectos 2018\my-drawer-ng\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:23: AA
PT: error: resource drawable/icon (aka org.nativescript.mydrawerng:drawable/icon) not found.

>FAILURE: Build failed with an exception.

>* What went wrong:
> Execution failed for task ':app:processDebugResources'.
> Failed to process resources, see aapt output above for details.

**Comment by ParVisual** [comment]
Some help about it? Thanks.

**Comment by albertobastos** [comment]
I had the same problem and, comparing my sources with the ones from a just-created app using a template, it looks like the drawable-xxx folders within App_Resources/Android have been moved and the bundler process now expects to find them in the new location.

Old location: app/App_Resources/Android/drawable-xxx
New location: app/App_Resources/Android/src/main/res/drawable-xxx

I moved those folders manually, removed and re-added the android platform, and the "build android" command no longer exits with error. I hope that helps you, too.

Probably something the "tns update" command should take into account, perhaps?
